apps/backend/core/configuration/validation/utils/http.py

```
@@ -0,0 +1,320 @@
+            return True, None, latency_ms
+        else:
+            error_msg = f"Unexpected status code: {response.status_code}"
+            try:
+                # Try to extract error message from response
+                error_data = response.json()
+                if "error" in error_data:
+                    if isinstance(error_data["error"], dict):
+                        error_msg += f" - {error_data['error'].get('message', '')}"
+                    else:
+                        error_msg += f" - {error_data['error']}"
+            except Exception:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)